clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -200,10 +200,60 @@ def isclose(a, b, rel_tol=1e-9, abs_tol=0.0):
     return list_of_regions
 
 
+def runApplyInverseDeformationField_SPM_standalone(target,
```

Comment:
Following PEP8 convention, this should be named `run_apply_inverse_deformation_field_spm_standalone` but I understand you did that to comply with `applyInverseDeformationField.m` Matlab file.

(For this one, I let you decide to modify or not)